experimental/Schemes/BlowupMorphism.jl

```diff
@@ -151,10 +161,60 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   inc_Z_trans = CoveredClosedEmbedding(Y, I_trans, 
                                        covering=simplified_covering(Y), # Has been set by the previous call
                                        check=false)
-  inc_cov = covering_morphism(inc_Z_trans)
+  inc_dom_cov = covering_morphism(inc_Z_trans)
+  inc_cod_cov = covering_morphism(inc)
 
   Z_trans = domain(inc_Z_trans)
   pr_res = restrict(projection(p), inc_Z_trans, inc)
+
+  if has_attribute(p, :isomorphism_on_open_subset)
+    OOX = OO(X)
+    OOY = OO(Y)
+    p_iso = isomorphism_on_open_subset(p)
+    U = domain(p_iso)::PrincipalOpenSubset
+    V = codomain(p_iso)::PrincipalOpenSubset
+    V_amb = ambient_scheme(V)
+    U_amb = ambient_scheme(U)
+
+    # Given all the refinements that potentially had to be done, we have 
+    # to do the following.
+    #  - Find a `patch` `U_sub` of the `domain` of `inc_dom_cov` for which 
+    #    inc_cod : U_sub -> W has a codomain `W` which has `U_amb` as an 
+    #    ancestor. Since `U_amb` is one of the `affine_charts` of `X`, this will work. 
+    #  - pr_res : U_amb -> V_amb has some codomain such that there exists 
+    #    an ancestor `VV` in the `domain` of `inc_dom_cov` such that 
+    #    inc_dom : VV -> W' has a codomain with `V_amb` as an ancestor. 
+    #  - outside the `complement_equation`s of `U` and `V` the projection 
+    #    was an isomorphism. Then the restriction of `pr_res` to those 
+    #    complements in `U_sub` and `V_sub` also is.
+    k = findfirst(x->has_ancestor(y->y===U_amb, codomain(inc_dom_cov[x])), patches(domain(inc_dom_cov)))
+    U_sub = patches(domain(inc_dom_cov))[k]
+    iso_U_sub = _flatten_open_subscheme(U_sub, domain(inc_dom_cov[U_sub]))
```

Comment:
what does this do and why do you do it?

Reply:
line 192 is deprecated and superfluous. Did you mean that? In that case: Thanks for catching!